### PR TITLE
build: text utility component

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
image
image
image

### CODE REVIEW

File: components/text-utility/text-utility.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    margin-bottom: 1rem;
+}
+.text-align p{
+    font-size: 1.5rem;
+ }
+ .text-align{
+     height: 100%;
+     width: 100%;
+ }
+ .centre-align{
```
Comment (purvasharmadev) [inline components/text-utility/text-utility.css:29]: Sugesstion: write class names descriptively.
for ex follow any naming convention, and if not you can always try writing class names as 
attribute-property for ex. align-text is more descriptive than text-align.

**Review by purvasharmadev — APPROVED** [review]
everything looks good to go! Nice work!!

File: components/text-utility/text-utility.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+ .right-align{
+     margin-left: 50rem;
+ }
+ .paragraph-lg{
+     font-size: 2rem;
+ }
+ .paragraph-rg{
+     font-size: 1.5rem;
+ }
```
Comment (bhushannaikaroskar) [inline components/text-utility/text-utility.css:43]: You class names are all good but I would suggest you to use some widely used size names like lg - for large, xl - for extra large. Here while reading I am unable to understand what exactly is rg. Please use relevant name. This is a suggestion tough you can correct me if I am missing something.

**Review by bhushannaikaroskar — APPROVED** [review]
Hey subham, good class naming didn't find much to correct anything. I have given some comments do reflect on it. For now approving.

**Review by arjun-computer-geek — APPROVED** [review]

File: components/text-utility/text-utility.css (hunk 1 above)
Comment (Shubhamkr2610) [reply components/text-utility/text-utility.css:29]: Sure. I'll refactor the classname.

File: components/text-utility/text-utility.css (hunk 2 above)
Comment (Shubhamkr2610) [reply components/text-utility/text-utility.css:43]: Sure. I'll  take care of it and refactor  it

**Pull request merged**